keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -206,11 +207,17 @@ while making routing decisions. It does not offer strict routing guarantees.
 
 The field will support the following initial values:
 
+* `PreferSameZone`: Indicates a preference for routing traffic to endpoints in
+  the same zone as the client.
 
-* `PreferClose`: Indicates a preference for routing traffic to endpoints that
-  are topologically proximate to the client. The interpretation of
-  "topologically proximate" may vary across implementations and could encompass
-  endpoints within the same node, rack, zone, or even region.
+The following value was proposed by a previous iteration of this KEP but will be
+deprecated in favor of `PreferSameZone`. It can still be used as an alias for
+`PreferSameZone` to ensure backward compatibility.
+
+* `PreferClose`: (Deprecated) Indicates a preference for routing traffic to
+  endpoints that are topologically proximate to the client. The interpretation
+  of "topologically proximate" may vary across implementations and could
+  encompass endpoints within the same node, rack, zone, or even region.
```

Comment:
#5140 now deals with introducing `PreferSameZone`. So for KEP-4444, I think you should just update the description of `PreferClose` to be what you have above for `PreferSameZone`, and then add an explanation that despite the vagueness of the name, it really doesn't mean anything other than "same zone", and maybe mention that KEP-3015 is introducing a new name for it?

Reply:
Yep updated the description.

I've added a reference to the other "An alternative definition of `PreferClose`" that you suggested for complete context.